Sources/Recycling/GridViews/Ads/Cell/AdsGridViewCell.swift

```diff
@@ -17,63 +17,63 @@ public protocol AdsGridViewCellDelegate: AnyObject {
 public class AdsGridViewCell: UICollectionViewCell {
     // MARK: - Internal properties
 
-    private static let titleHeight: CGFloat = 20.0
-    private static let titleTopMargin: CGFloat = 3.0
-    private static let bottomMargin: CGFloat = 15.0
-    private static let subtitleHeight: CGFloat = 17.0
-    private static let subtitleTopMargin: CGFloat = 6.0
-    private static let accessoryHeight: CGFloat = 14.0
-    private static let margin: CGFloat = 8.0
-    private static let cornerRadius: CGFloat = 8.0
-    private static let imageDescriptionHeight: CGFloat = 35.0
-    private static let iconSize: CGFloat = 23.0
-
-    private lazy var imageBackgroundView: UIView = {
+    public static let titleHeight: CGFloat = 20.0
```

Comment:
I guess all the modifiers changed to `public` in this file could be made `internal`. Or do we need them in the app?